test/rxd/test_currents.py

```diff
@@ -80,7 +80,7 @@ def test_currents_cvode(model_pump):
     h.continuerun(10)
     if not save_path:
         max_err = compare_data(data)
-        assert max_err < tol
+        assert max_err < 1e-8  # tol
```

Comment:
Rather than just new fixed value, should it be an if because it only was observed on an apple M1?

Reply:
next commit makes uses 1e-8 only for apple arm64.